src/main/java/org/opentripplanner/routing/bike_rental/BikeRentalStation.java

```diff
@@ -99,4 +102,17 @@ public BikeRentalStation clone() {
     public String getName() {
         return name.toString(locale);
     }
+
+    public static class RentalUris {
```

Comment:
Avoid using `public static` classes. Also add some relevant documentation, steal from the spec :-)

Reply:
The class has been moved to the top level, with a short javadoc and a link to the spec added.